gtsfm/averaging/translation/averaging_creta.py

```diff
@@ -0,0 +1,119 @@
+import os
+import time
+from typing import Dict, List, Optional, Tuple
+
+import numpy as np
+from scipy.io import savemat, loadmat
```

Comment:
let's import only the module, as `import scipy.io`, instead of importing the function.

https://google.github.io/styleguide/pyguide.html#22-imports